code/game/objects/items/weapons/melee/melee_misc.dm

```diff
@@ -511,6 +525,51 @@
 	S.melee_attack_chain(user, target)
 	target.Weaken(power)
 
+/datum/enchantment/time_slash
+	name = "temporal"
+	desc = "this blade will slice faster but weaker, and will curse the target, slashing them a few seconds after they have not been swinged at for each hit"
+	power = 15 // This should come out to 40 damage per hit. However, delayed.
+
+/datum/enchantment/time_slash/on_apply_to_blade(obj/item/melee/spellblade)
+	spellblade.force /= 2
+
+/datum/enchantment/time_slash/on_hit(mob/living/target, mob/living/user, proximity, obj/item/melee/spellblade/S)
+	user.changeNext_move(CLICK_CD_MELEE * 0.5)
+	. = ..()
+	if(!.)
+		return
+	target.apply_status_effect(STATUS_EFFECT_TEMPORAL_SLASH, power)
+
+/obj/effect/temp_visual/temporal_slash
+	name = "temporal slash"
+	desc = "A cut through spacetime"
+	icon = 'icons/obj/projectiles.dmi'
+	icon_state = "arcane_barrage"
+	layer = FLY_LAYER
+	plane = GRAVITY_PULSE_PLANE
+	appearance_flags = PIXEL_SCALE|LONG_GLIDE
+	duration = 0.5 SECONDS
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT // Let us not have this visual block clicks
+	/// Who we are orbiting
+	var/target
+	/// A funky colour matrix to recolor the slash to
+	var/list/funky_colour_matrix = list(0.4,0,0,0, 0,1.1,0,0, 0,0,1.65,0, -0.3,0.15,0,1, 0,0,0,0)
```

Comment:
```suggestion
	/// A funky color matrix to recolor the slash to
	var/list/funky_color_matrix = list(0.4,0,0,0, 0,1.1,0,0, 0,0,1.65,0, -0.3,0.15,0,1, 0,0,0,0)
```
Use color here instead of colour, please.